apis/basic/composition.yaml

```diff
@@ -531,3 +531,28 @@ spec:
                 patchSetName: network-id
               - type: PatchSet
                 patchSetName: region
+
+          - name: securityGroupRuleRedis
+            base:
+              apiVersion: ec2.aws.upbound.io/v1beta1
+              kind: SecurityGroupRule
```

Comment:
Can we start using the new Resource SecurityGroupIngressRule ? This resource using the real AWS identifier sgr-xxxxxx

Reply:
updated, and also added SecurityGroupIngressRule for Memcached.